### PR TITLE
tag v1.0.8

### PR DESCRIPTION
**Pull request by ulbqb**
## Description

For tagging v1.0.8

### CODE REVIEW

**Review by Mdaiki0730 — APPROVED** [review]

**Review by kokeshiM0chi — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,63 @@
...
+*Dec 27, 2022*
+
+* Update the default value of DefaultMaxTolerableByzantinePercentage
+* Fix Validators of RPC implementation in Ostracon
+* Add zerolog based rolling log system
+* Improve many components
+* Fix many bugs
+
+### BREAKING CHANGE
+- [rpc] [\#487](https://github.com/line/ostracon/pull/487) Fix Validators of RPC implementation in Ostracon
```
Comment (Mdaiki0730) [inline CHANGELOG.md]: Isn't it 478?

Comment (ulbqb) [reply CHANGELOG.md]: It already fixed in latest commit.

**Comment by Mdaiki0730** [comment]
[497](https://github.com/line/ostracon/pull/497) isn't included?

File: CHANGELOG.md (hunk 1 above)
Comment (Mdaiki0730) [reply CHANGELOG.md]: I overlooked it. I'm sorry🙇

**Review by Mdaiki0730 — APPROVED** [review]
LGTM

**Review by kokeshiM0chi — APPROVED** [review]

**Pull request merged**